03-task-manager/starter/ quizAnswers2.txt

```diff
@@ -0,0 +1,16 @@
+1. In this lesson, you created a middleware function called asyncWrapper. Why?
+- The main reason is to refactor out the try-catch block that was repeated in each task-related CRUD function.
```

Comment:
I haven't viewed the tutorial you're following, but in my opinion, the purpose is moreso that the `asyncWrapper` middleware helps to convert from Express's callback-based model for asynchrony to the Promise / async / await model. It handles errors from rejected promises, which does allow to avoid repetitive error handling code.